zerver/openapi/zulip.yaml

```diff
@@ -6492,7 +6492,7 @@ paths:
                         description: |
                           URL for the user's avatar.
 
-                          **Changes**: New in Zulip 2.1.0.
+                          **Changes**: New in Zulip 2.1.
```

Comment:
We changed our version-numbering scheme with Zulip 3.0, so that "Zulip 2.1" is actually parallel to "Zulip 3", and the parallel to "Zulip 3.0" is "Zulip 2.1.0".

That is: before Zulip 3.0, we made feature releases by incrementing the first or second number (and resetting later numbers), and bugfix releases by incrementing the third number. Starting with Zulip 3.0, a feature release increments the first number (and resets the second), a bugfix release increments the second, and there is no third number.

It looks like we have actually made API changes in a bugfix release, namely in 3.1:
https://zulip.com/api/changelog#changes-in-zulip-31
so for the new-style scheme, it probably makes sense to consistently include both numbers. As long as we're doing that and saying "Zulip 3.0", I think we should say "Zulip 2.1.0" to match.

In particular, that will avoid reinforcing an impression (which one could easily get from the numbers) that 2.1 was a patch/bugfix release on 2.0 just like 3.1 or 4.1 or 5.1 relate to 3.0 or 4.0 or 5.0 respectively.

Reply:
This looks great. I don't have strong feelings on the "Changes for Zulip 2.1" style headings; I'm fine with leaving things as-is or your starting a chat.zulip.org thread to help figure out a decision there.